shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java

```diff
@@ -72,100 +65,72 @@ public class ManagedBridgeController implements Reconciler<ManagedBridge>,
     @Inject
     OperatorMetricsService metricsService;
 
+    @Inject
+    ManagerClient managerClient;
+
     @Override
     public UpdateControl<ManagedBridge> reconcile(ManagedBridge managedBridge, Context context) {
 
         LOGGER.info("Reconciling ManagedBridge: '{}' in namespace '{}'",
                 managedBridge.getMetadata().getName(),
                 managedBridge.getMetadata().getNamespace());
 
-        ManagedBridgeStatus status = managedBridge.getStatus();
-
-        if (!status.isReady() && isTimedOut(status)) {
-            //TODO - Callback to Manager will be added in https://issues.redhat.com/browse/MGDOBR-1267
-            //            notifyManagerOfFailure(bridgeIngress,
-            //                                   new ProvisioningTimeOutException(String.format(ProvisioningTimeOutException.TIMEOUT_FAILURE_MESSAGE,
-            //                                                                                  bridgeIngress.getClass().getSimpleName(),
-            //                                                                                  bridgeIngress.getSpec().getId())));
-            status.markConditionFalse(ConditionTypeConstants.READY);
-            return UpdateControl.updateStatus(managedBridge);
-        }
-
-        Secret secret = managedBridgeService.fetchBridgeSecret(managedBridge);
+        try {
+            ManagedBridgeStatus status = managedBridge.getStatus();
 
-        if (secret == null) {
-            LOGGER.info("The Secret for the ManagedBridge with id '{}' has been not created yet.",
-                    managedBridge.getSpec().getId());
-            if (!status.isConditionTypeFalse(ConditionTypeConstants.READY)) {
-                status.markConditionFalse(ConditionTypeConstants.READY);
-            }
-            if (!status.isConditionTypeFalse(ManagedBridgeStatus.SECRET_AVAILABLE)) {
+            Secret secret = managedBridgeService.fetchBridgeSecret(managedBridge);
+            if (secret == null) {
                 status.markConditionFalse(ManagedBridgeStatus.SECRET_AVAILABLE);
```

Comment:
A general comment (before I review any more).

We had proposed to use `String` constants in [`DefaultConditions`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/DefaultConditions.java) in `infra-v2` so that we can reference the same constants throughout the entire codebase. WDYT? (Feel free to rename constants prefixed `CP_` and their values as you see fit).

Reply:
done